extensions/blocks/map/settings.js

```diff
@@ -26,7 +26,7 @@ export const settings = {
 	),
 	category: 'jetpack',
 	keywords: [
-		_x( 'map', 'block search term', 'jetpack' ),
+		_x( 'maps', 'block search term', 'jetpack' ),
```

Comment:
This one is interesting, as the trick with "map" and "maps" matching because one word fully contains the other only works in the English (possibly in some other languages too but it's not an universal rule). 

Should we just list both variants as there is no longer a keyword limit?

Reply:
> there is no longer a keyword limit

Is that a new change in Core? We may need to keep only 3 for now since we support WP 5.2 for a bit longer in Jetpack.

Reply:
the limit was removed in February 2019 - https://github.com/WordPress/gutenberg/pull/13848 (follow the discussion as it brings more context than just the description)

Reply:
That's good, Gutenberg 5.1 was released a while ago now so that's definitely something we can use in Jetpack today.